src/index.ts

```diff
@@ -64,23 +67,31 @@ export class YoutubeTranscript {
   /**
    * Fetch transcript from YTB Video
    * @param videoId Video url or video identifier
-   * @param config Get transcript in a specific language ISO
+   * @param config Specify a specific language ISO code or a proxy URL
    */
   public static async fetchTranscript(
     videoId: string,
     config?: TranscriptConfig
   ): Promise<TranscriptResponse[]> {
+    const axiosConfig = {
+      headers: {
+        ...(config?.lang && { 'Accept-Language': config.lang }),
+        'User-Agent': USER_AGENT,
+      },
+      // Add proxy configuration if provided
+      ...(config?.proxy && parseProxyURL(config.proxy)),
```

Comment:
I don't think this will work -- the proxy config needs to be in an object `proxy`:

`...(config?.proxy && { proxy: parseProxyURL(config.proxy) }),`